extensions/azurehybridtoolkit/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "azurehybridtoolkit",
+  "displayName": "%displayName%",
+  "description": "%description%",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "engines": {
+    "vscode": "*",
+    "azdata": "*"
+  },
+  "main": "./out/main",
+  "activationEvents": [
+    "*"
+  ],
+  "contributes": {
+    "commands": [
+      {
+        "command": "hybridtoolkit.openNotebooks",
+        "title": "%title.openNotebookFolder%",
+        "icon": {
+          "dark": "resources/dark/placeholder_logo_jupyterhub.svg",
```

Comment:
Not sure what you need icons for - this isn't currently set up to show anywhere that would display an icon.